databricks/koalas/series.py

```diff
@@ -648,6 +648,69 @@ def notnull(self):
 
     notna = notnull
 
+    def fillna(self, value=None, axis=None, inplace=False):
+        """Fill NA/NaN values.
+
+        Parameters
+        ----------
+        value : scalar, dict, Series
```

Comment:
i think we should just follow pandas here and drop dict/Series. This is a Series, so there's only one column really.

Reply:
@rxin 
[Pandas doc](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.fillna.html) says that it supports `dict` & `Series`. 
But I think the doc is a copy paste of `DataFrame.fillna`. 

Anyway, we can remove them from the docstring.